cms/static/js/certificates/views/certificate_details.js

```diff
@@ -0,0 +1,70 @@
+// Backbone Application View: Certificate Details
+
+define([
+    'js/views/baseview', 'underscore', 'js/certificates/models/signatory', 'js/certificates/views/signatory_details', 'gettext', 'underscore.string'
+],
+function(BaseView, _, Signatory, SignatoryDetails, gettext, str) {
+    'use strict';
+    var CertificateDetailsView = BaseView.extend({
+        tagName: 'div',
+        events: {
+            'click .edit': 'editCertificate',
+            'click .show-details': 'showDetails',
+            'click .hide-details': 'hideDetails'
+        },
+
+        className: function () {
+            // Determine the CSS class names for this model instance
+            var index = this.model.collection.indexOf(this.model);
```

Comment:
@asadiqbal08 -- FYI on this unused variable (scoped to function) -- did you intend to utilize it for something?  It could be a holdover from the group configurations bootstrap we did early-on.